src/views/EditorView/EditorContainer/EditorContainer.tsx

```diff
@@ -105,7 +112,7 @@ const EditorContainer: React.FC<IProps> = ({windowSize, activeImageIndex, images
             <div className="EditorWrapper"
                 onMouseDown={() => ContextManager.switchCtx(ContextType.EDITOR)}
             >
-                <EditorTopNavigationBar/>
+                {projectType === ProjectType.OBJECT_DETECTION && <EditorTopNavigationBar/>}
```

Comment:
jak w JSXie robisz takie warunki, to warto dodać props 'key' do każdego w elementów sąsiadujących na tym samym poziomie zagnieżdżenia. wtedy React wie lepiej jak porównywać elementy w virtualnym DOM.
albo inaczej: zawsze warto dodawać props 'key' :)